src/api/firebase.js

```diff
@@ -71,15 +78,30 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 		dateNextPurchased: getFutureDate(daysUntilNextPurchase),
 		name: itemName,
 		totalPurchases: 0,
+		checked: false,
```

Comment:
I like how this little "checked" guy 👦 is running the show throughout this issue!